src/appengine/private/templates/error-403.html

```diff
@@ -36,7 +36,8 @@
   {% if message %}
     <p>{{message}}</p>
   {% endif %}
-  <p>Please contact {{contact_string}}.</p>
+  <!--TODO(pgrace) remove this comment in Jan 2024 when we have moved to Buganizer -->
```

Comment:
Hmm...This is a little problematic because it assumes the user is from Chrome. Maybe specify if this website is clusterfuzz.com - please confirm you have bene manually CCed.
@oliverchang WDYT?

Reply:
ahh
would using is_chromium (https://github.com/google/clusterfuzz/blob/65b5ca95435835aeeb287b723452650a81d742ad/src/clusterfuzz/_internal/base/utils.py#L915) be good enough to make it only show up for chrome?

Reply:
+1 yep. We can pipe it through the template similar to here: https://github.com/google/clusterfuzz/blob/65b5ca95435835aeeb287b723452650a81d742ad/src/appengine/handlers/upload_testcase.py#L247

Reply:
done! thank you both!